src/components/modals/BtcSendModal.vue

```diff
@@ -479,6 +485,11 @@ export default defineComponent({
                             value: requiredInputs.value.changeAmount,
                         },
                     } : {}),
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore
+                    fiatCurrency: fiatCurrency.value,
```

Comment:
Instead of the usage of a new variable, just to assign it here, you can use the store here directly, without the getter in between:

```suggestion
                    fiatCurrency: useAccountStore().state.activeCurrency,
```